src/core/Akka.Cluster/Configuration/Cluster.conf

```diff
@@ -60,7 +60,9 @@ akka {
     # * if it is set to a duration the `AutoDowning` provider is with the configured downing duration
     #
     # If specified the value must be the fully qualified class name of a subclass of
-    # `akka.cluster.DowningProvider` having a public one argument constructor accepting an `ActorSystem`
+    # `akka.cluster.DowningProvider` having two argument constructor:
```

Comment:
Documented API changes, in case a user has a custom `IDowningProvider` implementation. I have _never seen one_ in the wild, which is why I think this breaking API change is probably safe for a v1.4.39 release.